### DISCUSSION TITLE
Referencing extensions in the core schema

### DISCUSSION THREAD

**Discussion by shuyag** [opensupplyhub/supplychaindata.exchange#19] [General]
Should a reference to an extension be referenced in the core schema as an optional field? (i.e. add in a name field into every extension as required) 

Would it be referenced in the location part of the schema, or the organization? Might be okay for both and have it be optional, depending on which one the extension is more relevant to.

**Comment by scrthq** [comment]
@shuyag I'm catching up on the intent of extensions, but if we are intending to allow arbitrary extension of the schema by others (e.g. add their own relevant fields while still adhering to the SCDEX schema itself), one way of accomplishing that is by allowing additional arbitrary keys to be included in the schema. This would effectively allow extension without the core SCDEX schema needing to be aware of the additional fields at all.
